src/components/WalletInit/RestoreWallet/SaveReadOnlyWalletScreen.js

```diff
@@ -156,17 +156,18 @@ const SaveReadOnlyWalletScreen = (
     return i
   })
 
-  const generatePlates = async () => {
-    const {addresses, accountPlate} = await generateShelleyPlateFromKey(
-      publicKeyHex,
-      1,
-      networkId,
-    )
-    setPlate({addresses, accountPlate})
-  }
-
   useEffect(() => {
+    const generatePlates = async () => {
+      const {addresses, accountPlate} = await generateShelleyPlateFromKey(
+        publicKeyHex,
+        1,
+        networkId,
+      )
+      setPlate({addresses, accountPlate})
+    }
+
     generatePlates()
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I guess it complains because `publicKeyHex` and `networkId` come from a prop? If that's the case, maybe we should just add them to the deps list.